### PR TITLE
Updated Problem JSON schema

### PR DESCRIPTION
**Pull request by whiskeysierra**
Partially relates to #120

### CODE REVIEW

File: common-data-objects/CommonDataObjects.md [hunk 1]
```diff
@@ -117,6 +116,7 @@ Here is the definition in Open API 2.0 YAML format in the case you don't need an
            An absolute URI that identifies the problem type.  When dereferenced,
            it SHOULD provide human-readable documentation for the problem type
            (e.g., using HTML).
+         default: 'about:blank'
          example: http://httpstatus.es/503
```
Comment (jhorstmann) [inline common-data-objects/CommonDataObjects.md]: Maybe add a better example that does not just repeat the status code but adds meaning to the error, like `http://example.com/problems/out-of-stock`

Comment (whiskeysierra) [inline common-data-objects/CommonDataObjects.md]: I updated the schema (and externalized it).

**Comment by ePaul** [comment]
:+1:

**Comment by fmueller** [comment]
👍

**Pull request merged**